templates/user/dashboard/feeds.tmpl

```diff
@@ -1,125 +1,149 @@
 <div id="activity-feed" class="flex-list">
 	{{range .Feeds}}
+		{{$isPrivateView := and (not $.IsAdmin) .IsPrivate (ne $.ContextUser.ID $.SignedUserID) (not $.IsOrganizationMember)}}
```

Comment:
`$.IsOrganizationMember` only available in `{Host}/org/{OrgName}/dashboard`, but this template is also used in `{Host}/{Username}?tab=activity`. Then you will get this:
In dashboard you can see the details of private actions, as `$.IsOrganizationMember` is true and `$isPrivateView` is false.
![image](https://github.com/go-gitea/gitea/assets/18380374/a1491836-97d4-4220-9f58-84523a9b2148)
But in user's public activity page, it will be hidden, as `$.IsOrganizationMember` is empty(false) and `$isPrivateView` is true.
![image](https://github.com/go-gitea/gitea/assets/18380374/0ff8acc3-b0d7-4348-be91-88557c6bfe4a)

Reply:
Its not correct:

Own public activity:
![image](https://github.com/go-gitea/gitea/assets/72873130/ecfce10e-6590-4407-b3a5-f1921bce3016)

other users activity:
![image](https://github.com/go-gitea/gitea/assets/72873130/8a5557fc-b8d5-411b-938f-87b025b8a4a3)

own dashboard:
![image](https://github.com/go-gitea/gitea/assets/72873130/d10b999d-c712-4a46-9c97-3e2293a40e40)

Reply:
its fixed